webapp/src/pages/Settings.tsx

```diff
@@ -253,14 +275,38 @@ const Settings: React.FC<Props> = ({ open, onClose }) => {
       <DialogTitle id="config-dialog-title">
         <Box display="flex" justifyContent="space-between" alignItems="center">
           <Typography variant="subtitle1">
-            View and edit certain fields from your config file. Once your
-            changes are saved, expect some delays for recomputing the affected
-            tasks.
+            View and edit your config. Once your changes are saved, expect some
+            delays for recomputing the affected tasks.
```

Comment:
I shortened that since it's not only "certain fields" anymore, but it's still very long for a title.
![image](https://github.com/ServiceNow/azimuth/assets/8386369/20237944-4818-4a9e-99dd-3512e4ed2b29)

I think the second sentence is redundant with the "These changes may trigger some time-consuming computations." that we get next to the Apply button when we make changes that trigger recompute. Can I remove it?
```suggestion
            View and edit your config.
```
Or even a simple quick title?
```suggestion
            Config
```

Reply:
Given that now it's also there the first time you launch Azimuth without a config, I would just go with `Configuration'.